build.sbt

```diff
@@ -62,12 +62,14 @@ lazy val channel = project
 lazy val monix = project
   .in(file("modules/streaming/monix"))
   .dependsOn(channel)
+  .dependsOn(server)
   .dependsOn(`internal-monix`)
   .settings(moduleName := "mu-rpc-monix")
 
 lazy val fs2 = project
   .in(file("modules/streaming/fs2"))
   .dependsOn(channel)
+  .dependsOn(server)
```

Comment:
mmm... Why do the `monix` and `fs2` modules would depend on the server?

Reply:
Because `monix` and `fs2` only contains the healthcheck service implementation part for `monix` and `fs2`, respectively. And they need the unary service implementation which is placed in `server`. It's a little bit confusing, the aim is to reduce the numbers of modules.

Reply:
I think the unary service and implementation can be located on the `channel` module.

Reply:
👍  I agree on putting it in the `channel` module